tests/helpers.py

```diff
@@ -378,15 +376,16 @@ def get_cephfs_data_pool_name():
     return out[0]['data_pools'][0]
 
 
+@retry(CommandFailed, tries=5, delay=3, backoff=1)
 def validate_cephfilesystem(fs_name):
```

Comment:
let's pass the cephfs object instead of the name so you won't have to init OCP() for the FS

Reply:
if we pass obj eventually fs is created from OCP side  so here validating from ocp side doesn't make sense you can correct me where am i wrong

Reply:
I see. I think the implementation of create_filesystem in helpers.py should be as done in create_ceph_block_pool so the function